modules/tls-profiles-kubelet-configuring.adoc

```diff
@@ -0,0 +1,131 @@
+// Module included in the following assemblies:
+//
+// * security/tls-profiles.adoc
+
+ifeval::["{context}" == "tls-security-profiles"]
+:tls:
+endif::[]
+
+[id="tls-profiles-kubelet-configuring_{context}"]
+= Configuring TLS profiles for the kubelet
+
+To configure a TLS security profile for the kubelet when it is acting as an HTTP server, create a `KubeletConfig` custom resource (CR) to specify a predefined or custom TLS profile for specific nodes. If a TLS profile is not configured, the default TLS profile is `Intermediate`. 
+
+ifdef::tls[]
+The kubelet uses its HTTP/GRPC server to communicate with the Kubernetes API server, which sends commands to pods, gathers logs, and run exec commands on pods through the kubelet.
+endif::[]
+
+.Sample `KubeletConfig` CR that configures the `Old` TLS security profile on worker nodes
+[source,yaml]
+----
+apiVersion: config.openshift.io/v1
+kind: KubeletConfig
+ ...
+spec:
+  tlsSecurityProfile:
+    old: {}
+    type: Old
+  machineConfigPoolSelector:
+    matchLabels:
+      pools.operator.machineconfiguration.openshift.io/worker: ""
+----
+
+You can see the ciphers and the minimum TLS version of the configured TLS security profile in the `kubelet.conf` file on a configured node.
+
+[IMPORTANT]
+====
+The kubelet does not support TLS `1.3` and because the `Modern` profile requires TLS `1.3` it is not supported. The kubelet converts the `Modern` profile to `Intermediate`.
+
+The kubelet also converts the TLS `1.0` of an `Old` or `Custom` profile to `1.1`, and TLS `1.3` of a `Custom` profile to `1.2`.
+====
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. Create a `KubeletConfig` CR to configure the TLS security profile:
++
+.Sample `KubeletConfig` CR for a `Custom` profile
+[source,yaml]
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: KubeletConfig
+metadata:
+  name: set-kubelet-tls-security-profile
+spec:
+  tlsSecurityProfile:
+    type: Custom <1>
+    custom: <2>
+      ciphers: <3>
+      - ECDHE-ECDSA-CHACHA20-POLY1305
+      - ECDHE-RSA-CHACHA20-POLY1305
+      - ECDHE-RSA-AES128-GCM-SHA256
+      - ECDHE-ECDSA-AES128-GCM-SHA256
+      minTLSVersion: VersionTLS11
+  machineConfigPoolSelector:
+    matchLabels:
+      pools.operator.machineconfiguration.openshift.io/worker: "" <4>
+----
++
+<1> Specify the TLS security profile type (`Old`, `Intermediate`, or `Custom`). The default is `Intermediate`.
+<2> Specify:
+* `old: {}`
+* `intermediate: {}`
+* `custom:`
+<3> For the `custom` type, specify a list of TLS ciphers and minimum accepted TLS version.
+<4> Optional: Specify the machine config pool label for the nodes you want to apply the TLS security profile. 
+
+. Create the `KubeletConfig` object:
++
+[source,terminal]
+----
+$ oc create -f <filename>
+----
+
+.Verification
+
+. After the nodes are in the `Ready` state, verify that the profile is set on a configured node:
```

Comment:
Since you still have this as just one step, it should be an unnumbered bullet

Reply:
An unnumbered bullet with substeps? Looks odd. 
```

* After the nodes are in the `Ready` state, verify that the profile is set on a configured node:

.. Start a debug session for a configured node:
+
[source,terminal]
----
$ oc debug node/<node_name>
----
```

Reply:
Yeah, but that's how you would do it to follow the guidelines. My first suggestion was to make each of those a/b/c/d steps their own top-level step